Sources/Responses/BytesRepresentable.swift

```diff
@@ -1,3 +1,18 @@
+import Foundation
+
+// public extension Equatable where Self: BytesRepresentable {
+//   public static func == (lhs: Self, rhs: Self) -> Bool {
+//     return lhs.toBytes == rhs.toBytes
+//   }
+// }
+
```

Comment:
I tried to extend the Equality operator conditionally to types that conform to BytesRepresentable, but this did not work either.